src/base/misc/fatfs.c

```diff
@@ -506,7 +506,7 @@ int read_boot(fatfs_t *f, unsigned char *b)
 
     build_boot_blk(f, b);
 
-    memcpy(b + 0x03, "DOSEMU10", 8);
+    memcpy(b + 0x03, "IBM  3.3", 8);
```

Comment:
Yes but you can run the programs like norton diskeditor
and analyse the boot sector with it. I'd even change it to
"DOSEMU20". :)
In any case, this is entirely up to you.